components/HomeHero/Strapline/Strapline.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from './Strapline.module.scss';
+import type { StraplineProps } from './types';
+
+export const Strapline = ({ icon, text }: StraplineProps) => (
+    <div className={styles.strapline}>
+        <strong className={styles.strapline__icon}>
+            {icon === 'thumbs up' ? '👍 Yes!' : '👎 No!'}
+        </strong>
```

Comment:
I'm not sure if the preview deployment for this site is correct - instead of showing the strapline component here, the following gets rendered instead.

![image](https://user-images.githubusercontent.com/18164832/155689429-8ef385c3-b1d2-492f-a808-920557568852.png)

I'm probably just reviewing a previous commit from this branch, I will review the latest work as well :)

Reply:
I split up the work on the `sections` branch in these 'smaller' branches. The goal of these MR is just to add the components but they aren't used yet.